src/app/features/cases/components/organisms/TableCases/TableCases.tsx

```diff
@@ -10,24 +10,27 @@ import { useCases } from "app/state/rest/config"
 
 type ButtonProps = {
   href: string
+  text: string
 }
-const OpenButton: React.FC<ButtonProps> = ({ href }) =>
+const OpenButton: React.FC<ButtonProps> = ({ href, text }) =>
   <ButtonLink to={href}>
-    <Button as="span" variant="textButton" iconSize={14} iconLeft={<ChevronRight />}>Open</Button>
+    <Button as="span" variant="textButton" iconSize={14} iconLeft={<ChevronRight />}>{ text }</Button>
   </ButtonLink>
 
 const columns = [
   { header:"Startdatum", minWidth: 100 },
   { header:"Einddatum", minWidth: 100 },
   { header:"Adres", minWidth: 300 },
-  { minWidth: 90 }
+  { minWidth: 120 },
+  { minWidth: 120 }
 ]
 
 const mapData = (data: API.Case) => [
   data.start_date ?? "-",
   data.end_date ?? "-",
   data.address.street_name ?? "-",
-  <OpenButton href={to("/cases/:id", { id: data.id })} />
+  <OpenButton href={to("/cases/:id", { id: data.id })} text="open" />,
+  <OpenButton href={to("/case-visits/:caseId", { caseId: data.id })} text="bezoeken" />
```

Comment:
Might be better to put both `OpenButton` components in the same column:
Only the last column is sticky on mobile.

Reply:
Ok. Will push.